pdbtools/pdb_tidy.py

```diff
@@ -143,11 +143,18 @@ def make_TER(prev_line):
     fmt_TER = "TER   {:>5d}      {:3s} {:1s}{:>4s}{:1s}" + " " * 53 + "\n"
 
     records = ('ATOM', 'HETATM')
-    ignored = ('TER', 'END ', 'END\n', 'CONECT', 'MASTER')
```

Comment:
Why remove the 'END ' and 'END\n' ? There must have been there for a reason

Reply:
Yes, because before those two cases were not addressing all cases. Now, we have only `END` but there is a `.strip()` at the correct `if`. Let me check I did not forget any `strip`.